### DISCUSSION TITLE
[Emergency] Fix 2FA security not available error. #1768797

### DISCUSSION THREAD

**Discussion by mvn-quannguyen3-dn-admin** [community/community#31434] [Other Feature Feedback, Questions, & Ideas]
A member of the organization is experiencing security issues 2FA is not available during login to the account, we tried to use the recovery code restore but also failed.
Due to the urgent nature of the work, please assist in fixing this issue as soon as possible.
We appreciate your prompt support, thanks.

**Comment by radar-blip** [comment]
Here is a link to [Github Support](https://support.github.com/request)
First column, first row: Account or Billing issues.
